app/models/manageiq/providers/autosde/storage_manager.rb

```diff
@@ -184,6 +184,7 @@ def connect(options = {})
     host       = options[:host] || address
     port       = options[:port] || self.port
     self.class.raw_connect(username, password, host, port).login
+    EmsRefresh.queue_refresh(ext_management_system)
```

Comment:
I paid attention to that, but I thought maybe for all those cases a refresh would fit well.

But if not, would it be preferable to do it 'after_save'?

Reply:
Ok, I understand that after_save is not option (I tried and saw an infinite loop...). Where would you recommend it then? I don't see anywhere else in the file where this could fit.